### ISSUE TITLE
Cross-container output only includes last component of filename

### ISSUE THREAD

**Issue by garrettheel** [benfred/py-spy#220]
When running `py-spy record --gil -p somepid --duration 10 --output /tmp/out.svg` with `py-spy==0.3.2`, I'm seeing that the resulting output contains only the filename, but never a leading path.

Here's an example stack that I'm seeing with `--format raw`:
```
_handle_and_close_when_done (baseserver.py:26);handle (ggevent.py:139);handle (base_async.py:41)
```

I would expect the output to be something like `handle (gunicorn/workers/ggevent.py:139)`, where the source of the file is much clearer. The example on the README suggests this should work and this has worked in the past with pyflame.

One possibly related piece of information is that `py-spy` is being run across containers here.

**Comment by benfred** [comment]
This seems to be an issue with how we are detecting short filenames across containers - will get a fix in soon

**Comment by benfred** [comment]
This should be fixed by #223

**Comment by garrettheel** [comment]
Tested this out and it fixes my issue. Looking forward to a release soon so we can take advantage of this. Thanks @benfred!
